### PR TITLE
fix(deps): don't pin request-promise-native and update renovate config

### PR DESCRIPTION
**Pull request by tripodsan**

### CODE REVIEW

File: .renovaterc.json [hunk 1]
```diff
@@ -26,10 +26,11 @@
       "excludePackagePatterns": [
         "^@adobe/"
       ],
+      "pin": false,
```
Comment (tripodsan) [inline .renovaterc.json]: I think this is wrong: 
https://docs.renovatebot.com/configuration-options/#pin

@trieloff do you know how to do this?

Comment (tripodsan) [reply .renovaterc.json]: maybe this: https://docs.renovatebot.com/presets-default/#donotpinpackageltarg0gt

**Comment by trieloff** [comment]
What’s the benefit of not pinning?

**Comment by tripodsan** [comment]
> What’s the benefit of not pinning?

actually, we only need it for `request-promise-native`, as their module loading messes up `request` when 2 libraries are installed in the same process.

so when helix-status has a `1.0.7` version and a parent has `1.0.8`, it messes up the bundle:

*works*
```
$npm ls request-promise-native
@adobe/helix-googledocs2md@1.3.6 /Users/tripod/codez/helix/helix-googledocs2md
├─┬ @adobe/helix-ops@1.0.1
│ └── request-promise-native@1.0.8  deduped
├─┬ @adobe/helix-status@7.0.0
│ └── request-promise-native@1.0.8  deduped
├─┬ @adobe/openwhisk-action-builder@2.3.1
│ └── request-promise-native@1.0.8  deduped
└── request-promise-native@1.0.8
```
*fails*
```
$ npm ls request-promise-native
@adobe/helix-googledocs2md@1.3.6 /Users/tripod/codez/helix/helix-googledocs2md
├─┬ @adobe/helix-ops@1.0.1
│ └── request-promise-native@1.0.8  deduped
├─┬ @adobe/helix-status@7.0.0
│ └── request-promise-native@1.0.7
├─┬ @adobe/openwhisk-action-builder@2.3.1
│ └── request-promise-native@1.0.8  deduped
└── request-promise-native@1.0.8
```

**Comment by tripodsan** [comment]
I will narrow the change and only _unpin_ `request-promise-native`.

also see https://docs.renovatebot.com/dependency-pinning/

**Pull request merged**